Netezos/Forging/IUnforge.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Netezos.Forging.Models;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Netezos.Forging
+{
+    public interface IUnforge
```

Comment:
I think there should be just one method 
````cs
public interface IUnforge
{
    Task<(string, IEnumerable<OperationContent>)> UnforgeOperationAsync(byte[] content);
}
````
Otherwise, we don't know which method to use (and you have already faced that problem when were writing tests).